Sources/TuistKit/Commands/TuistCommand.swift

```diff
@@ -119,33 +120,46 @@ public struct TuistCommand: AsyncParsableCommand {
             }
         }
 
+        let outputCompletion = {
+            WarningController.shared.flush()
+            outputLogfilePath(logFilePath)
+        }
+
         do {
-            defer { WarningController.shared.flush() }
             try await executeCommand()
+            outputCompletion()
         } catch let error as FatalError {
-            WarningController.shared.flush()
             errorHandler.fatal(error: error)
+            outputCompletion()
             _exit(exitCode(for: error).rawValue)
         } catch let error as ClientError where error.underlyingError is ServerClientAuthenticationError {
-            WarningController.shared.flush()
             // swiftlint:disable:next force_cast
             ServiceContext.current?.logger?.error("\((error.underlyingError as! ServerClientAuthenticationError).description)")
+            outputCompletion()
             _exit(exitCode(for: error).rawValue)
         } catch {
-            WarningController.shared.flush()
             if let parsedError {
                 handleParseError(parsedError)
             }
+
             // Exit cleanly
             if exitCode(for: error).rawValue == 0 {
                 exit(withError: error)
             } else {
                 errorHandler.fatal(error: UnhandledError(error: error))
+                outputCompletion()
                 _exit(exitCode(for: error).rawValue)
             }
         }
     }
 
+    private static func outputLogfilePath(_ logFilePath: AbsolutePath) {
+        // TODO:
+        // Once we introduce Noora, we should merge all the "completion" messages
+        // using the Noora's completion component.
```

Comment:
```suggestion
```

I'd personally not include small quality of live TODOs in the codebase. But we can also keep it, don't feel strongly about it.